packages/enzyme-adapter-react-helper/src/getAdapterForReactVersion.js

```diff
@@ -1,30 +1,40 @@
 import semver from 'semver';
 
+function isPreRelease(version) {
+  return semver.valid(version) && semver.SemVer(version).prerelease.length > 0;
```

Comment:
This is accessing semver internals, which may not be super great. Another option could be trying to regex the version number or something.

Reply:
i note that semver has an `includePrerelease` option that we could probably use in `intersects`, that defaults to false?

Separately, `semver. prerelease()` can be used to determine if it's a prerelease without using any internals.

Reply:
Ah, nice. I missed that one

Reply:
Also, we can definitely use `includePrerelease`, but it doesn't quite do what we want (as far as I can tell, in my testing).

Not sure exactly how `includePrerelease` affects `intersects`, but because a prerelease version is lower than its corresponding non-prerelease, they tend to match a version lower than we intend.